integration/spark-common/src/main/scala/org/apache/carbondata/events/Events.scala

```diff
@@ -62,6 +62,18 @@ trait DropTableEventInfo {
   val ifExistsSet: Boolean
 }
 
+/**
+ * event for show cache
+ */
+trait ShowTableCacheEventInfo {
+  val carbonTable: CarbonTable
+}
+
+trait ShowAllCacheEventInfo {
```

Comment:
This is not used anywhere. Please remove

Reply:
Done.